### DISCUSSION TITLE
MacOS build cache size.

### DISCUSSION THREAD

**Discussion by bader** [intel/llvm#8788] [Q&A]
@alexbatashev, @AlexeySachkov, do you know why MacOS build takes ~1.5G, whereas Linux build with shared libs takes ~350Mb?
See https://github.com/intel/llvm/actions/caches.
I think MacOS binaries are disposed after the build and we can try to optimize the size of the cached binaries. Can we use shared libs for MacOS build to save some space (assuming this is the main saver)?

**Comment by alexbatashev** [comment]
I wouldn't expect switching to dylib to change anythin about cache size. ccache saves cache of object files rather than final binaries. One thing that may have impact on the cache size is Apple universal binaries, but I don't know if this feature is enabled by default in recent versions of Xcode.
